### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by riadus**

### CODE REVIEW

File: RiadIMDB/IMDB.Common/StringExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace IMDB.Common
+{
+    public static class StringExtensions
+    {
+        public static bool IsNullOrEmpty(this string s) => string.IsNullOrEmpty(s);
```
Comment (arctouch-victorbotamedi) [inline RiadIMDB/IMDB.Common/StringExtensions.cs:5]: It feels weird to expect that you can call an extension method with a `null` value. But i understand the shortcut here..

File: RiadIMDB/IMDB.Domain/DTOs/MovieDto.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace IMDB.Domain.DTOs
+{
+    public class MovieDto
+    {
+        [JsonProperty("vote_count")]
+        public int VoteCount { get; set; }
```
Comment (arctouch-victorbotamedi) [inline RiadIMDB/IMDB.Domain/DTOs/MovieDto.cs:9]: It is possible to use a `NamingStrategySnakeCase`, so we don't have to declare a `JsonProperty` for each member.

File: RiadIMDB/IMDB.Domain/Services/LanguageService.cs [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+        public Language DeviceLanguage => GetLanguage(_deviceInformations.DeviceLanguage);
+
+        public Language CurrentLanguage { get; set; }
+
+        public Language? GetLanguage(string lang)
+        {
+            if (_languages.Any(x => x.ToString() == lang))
+            {
+                return _languages.FirstOrDefault(x => x.ToString() == lang);
```
Comment (arctouch-victorbotamedi) [inline RiadIMDB/IMDB.Domain/Services/LanguageService.cs:41]: You can call `_languages.FirstOrDefault(x => x.ToString() == lang)` directly here as if the condition is not met the return value is going to be `null`.

**Review by arctouch-victorbotamedi — COMMENTED** [review]
Nice work using dependency injection and handling possible errors.

There is just an issue where sometimes while scrolling the list an unknown error occurs.

File: RiadIMDB/IMDB.Domain/Services/LanguageService.cs (hunk 3 above)
Comment (riadus) [reply RiadIMDB/IMDB.Domain/Services/LanguageService.cs:41]: Indeed. I don't see why I added that useless extra check.

**Pull request merged**